astacus/coordinator/plugins/base.py

```diff
@@ -533,6 +533,36 @@ async def clear_delta(
         await cluster.wait_successful_results(start_results=start_results, result_class=ipc.NodeResult)
 
 
+def _prune_manifests(manifests: List[ipc.BackupManifest], retention: Retention) -> List[ipc.BackupManifest]:
+    manifests = sorted(manifests, key=lambda m: (m.start, m.end, m.filename), reverse=True)
```

Comment:
NB: Not sure if some later step makes assumptions about order -> sort even if we don't change the list here.